src/libraries/System.Private.CoreLib/src/System/TimeOnly.cs

```diff
@@ -175,6 +175,10 @@ private TimeOnly AddTicks(long ticks, out int wrappedDays)
         /// <param name="start">The starting time of day, inclusive.</param>
         /// <param name="end">The ending time of day, exclusive.</param>
         /// <returns>True, if the time falls within the range, false otherwise.</returns>
+        /// <remarks>
+        /// If the start and end are equal this method always returns false, meaning there is zero elapsed time between the two values.
+        /// If you wish to treat such cases as 24 hours, then first check for equality before calling this method.
```

Comment:
Why would it be 24 hours?  That would mean start and end were different days, in which case, why not 48 or 72 or ...? :-)

Reply:
The example cited in the linked issue was a store with hours from 00:00 to 00:00 meaning it was open 24-hours.  But you are correct, it could be any number of whole days, all of which would test true.  I updated the wording accordingly.